content/modules/rest/pages/managing-files.adoc

```diff
@@ -0,0 +1,174 @@
+= Manage Files
```

Comment:
What about naming all sections about generic endpoints "... API"?
Entities API
Files API
...

Reply:
sounds good. Will do it!